docs/what-we-look-at.md

```diff
@@ -0,0 +1,48 @@
+# What we look at
+
+Licensee works by taking a detected license file, and comparing the contents to a short list of known licenses.
+
+## Detecting the license file
+
+Licensee uses [a series of regular expressions](https://github.com/benbalter/licensee/blob/master/lib/licensee/project_files/license_file.rb#L15-L25) to score files in the project's root as potential license files. Here's a few examples of files that would be detected:
```

Comment:
Line numbers in relative version references? :) Ideally we should use a tag here (or ultimately, a commit hash).